docs/UserGuide.md

```diff
@@ -185,13 +185,13 @@ Format:`find n/NAME|e/EMAIL|p/PHONE_NUMBER|a/ADDRESS|r/REMARK|m/MEETING_DATE`<br
 * User input will be validated **only** for `find m/MEETING_DATE` based on the `DATE` format.
   * For other categories, user input does not have to conform to the corresponding parameter format, e.g. `find p/John`, however no persons will be matched.
 * Multiple keywords can be specified for tags **only**.
+  * Persons matching **at least** one of the specified tag keywords will be returned. e.g. `find t/PRUActiveCash t/friends` returns all persons with tags containing `PRUActiveCash` or `friends`.
```

Comment:
Maybe it would be good to shift the example down, to follow the existing format:

line 188: Persons matching at least one fo the specified tag keywords will be returned.

Line 194: `find t/PRUActiveCash t/friends` returns all persons with tags containing at least one of `PRUActiveCash` and `friends`

Reply:
I think the current format is better since the example follows from the explanation, if we split it there's a discontinuation between the explanation and example

Reply:
@Arixeyeion @Dethada I agree that the current commit highlights the use case of the tag searches as well